centrifuge-js/src/modules/pools.ts

```diff
@@ -943,7 +943,10 @@ export function getPoolsModule(inst: Centrifuge) {
             if (options?.createType === 'propose') {
               const proposalTx = api.tx.utility.batchAll([
                 api.tx.preimage.notePreimage(tx.method.toHex()),
-                api.tx.democracy.propose({ Inline: tx.method.hash }, api.consts.democracy.minimumDeposit),
+                api.tx.democracy.propose(
+                  { Lookup: [tx.method.hash, tx.method.encodedLength] },
```

Comment:
Did you test this call? I don't think it works as `Lookup` expects an object, i.e.
```suggestion
                  { Lookup: { hash: tx.method.hash, len: tx.method.encodedLength} },
```

Reply:
I tested and a tuple seemed to work: https://polkadot.js.org/apps/?rpc=wss://fullnode.development.cntrfg.com#/explorer/query/0x23cba13ab8577309b22854fca0bacb04555a1ac4c2834543ffc3c1d527538f6c

Reply:
Awesome, thanks!